csuntechlab/iSTART
# STR-210 -- FE: Updated allotted time to continue from 8 seconds to 5 seconds.

# Description
Updated allotted as seen in informational slide from 8 seconds to 5 seconds. Updated tooltip to reflect changes.
  
# Testing
1. Login through the Modules route.
2. Ensure on the informational slides, the time has changed from 8 seconds to 5 seconds wait time.
3. Check to see if the tooltip info has been changed to 5 seconds.
